packages/platform-android/native_modules.gradle

```diff
@@ -150,37 +153,44 @@ class ReactNativeModules {
     if (this.reactNativeModules != null) return this.reactNativeModules
     ArrayList<HashMap<String, String>> reactNativeModules = new ArrayList<HashMap<String, String>>()
 
-    def cmdProcess
     def npx = Os.isFamily(Os.FAMILY_WINDOWS) ? "npx.cmd" : "npx"
-    def command = "${npx} --quiet react-native config"
+    def command = "${npx} --quiet --no-install react-native config"
```

Comment:
Adding `--no-install` so we don't accidentally install `react-native` into local npm cache. Verified that it's already present in npx shipped with Node 8.

Reply:
Is it related to https://github.com/react-native-community/cli/issues/823?

Reply:
Not really sure, but it's preferable to run with `--no-install` anyway. Feel free to send a PR, as I don't want to mix iOS and Android in this one :)